Changelog.md

```diff
@@ -1,21 +1,23 @@
-### 0.7.3 (unreleased)
+### 0.7.3 (2020-10-07)
 
 Important Bugfixes:
  * Code Generator: Properly cleanup after copying dynamic-array to storage for packed types.
 
+
 Compiler Features:
+ * Code generator: Implemented events with function type as one of its indexed parameters.
+ * General: Option to stop compilation after parsing stage. Can be used with ``solc --stop-after parsing``
+ * Optimizer: Optimize ``exp`` when base is ``-1``.
  * SMTChecker: Support ``addmod`` and ``mulmod``.
  * SMTChecker: Support array slices.
  * SMTChecker: Support type conversions.
- * Optimizer: Optimize ``exp`` when base is -1.
- * Code generator: Implemented events with function type as one of its indexed parameters.
- * General: Option to stop compilation after parsing stage. Can be used with ``solc --stop-after parsing``
 
 
 Bugfixes:
- * Type Checker: Fix internal compiler error when calling `.push(<arg>)` for a storage array with a nested mapping.
+ * Fixed internal compiler errors for certain contracts involving the ``new`` expression.
+ * JSON AST: Fix internal error when using ``--ast-json`` on a function with memory arguments.
```

Comment:
Looks like `ABIEncoderV2` is an important factor too so I'd mention it. Apart from that I did a bit of quick testing and it it's a bit inconsistent but it does affect only memory arguments.
- Internal functions are fine
- Dynamic arrays are fine. But `string` and `bytes` are not.
- Dynamic arrays of `strings` are fine.
- Static arrays of structs are fine. Structs are not.

So my final suggestion:
```suggestion
 * JSON AST: Fix internal error when using ``--ast-json`` on a function with memory arguments in ABIEncoderV2 contracts.
```